src/Hazelcast.Net/Core/ClientVersion.cs

```diff
@@ -53,5 +57,33 @@ internal static string Version
                 return _clientVersion;
             }
         }
+
+        /// <summary>
+        /// Gets the "pure" version of this assembly.
+        /// </summary>
+        /// <remarks>
+        /// <para>Returns the pure version, ie major.minor.patch.</para>
+        /// </remarks>
+        internal static string PureVersion
+        {
+            get
+            {
+                if (_clientVersionPure != null) return _clientVersionPure;
+                var version = Version;
+                var pos0 = version.IndexOf('.');
```

Comment:
What are all these github check actions warning messages ? Do we have a plan to solve them? 
```
Check warning on line 73 in src/Hazelcast.Net/Core/ClientVersion.cs

GitHub Actions
/ Build PR (windows-latest)
src/Hazelcast.Net/Core/ClientVersion.cs#L73
'string.IndexOf(char)' has a method overload that takes a 'StringComparison' parameter. Replace this call in 'Hazelcast.Core.ClientVersion.PureVersion.get' with a call to 'string.IndexOf(char, System.StringComparison)' for clarity of intent.
```

Reply:
say the original `IndexOf` method only accepted 1 arg, and then a new version of .NET introduces an overload that accepts 2 args, and they recommend you use 2 args, so now we have a warning - they are harmless but I would die to fix them. This is a TECH_DEBT task that I need to create in Jira. Will do.